packages/aws-lambda/.gitignore

```diff
@@ -3,3 +3,4 @@ instana-aws-lambda.tgz
 instana-aws-lambda-auto-wrap.tgz
 instana-serverless.tgz
 npm-debug*
+include/
```

Comment:
I think we should actually commit the extension binary to the repo instead of ignoring it, see below.